util/envDynawo.sh

```diff
@@ -1412,27 +1450,19 @@ deploy_dynawo() {
   current_dir=$PWD
   mkdir -p $DYNAWO_DEPLOY_DIR || error_exit "Impossible to create $DYNAWO_DEPLOY_DIR."
   cd $DYNAWO_DEPLOY_DIR
-  mkdir -p 3rdParty/sundials/lib
-  mkdir -p 3rdParty/adept/lib
-  mkdir -p 3rdParty/suitesparse/lib
-  mkdir -p 3rdParty/nicslu/lib
-  mkdir -p extraLibs/BOOST/lib/
-  mkdir -p extraLibs/XERCESC/lib/
-  mkdir -p extraLibs/LIBARCHIVE/lib/
-  mkdir -p extraLibs/LIBZIP/lib
-  mkdir -p extraLibs/LIBXML/lib
-  mkdir -p extraLibs/LIBIIDM/lib
-  cp -P $DYNAWO_SUNDIALS_INSTALL_DIR/lib*/*.* 3rdParty/sundials/lib/
-  cp -P $DYNAWO_ADEPT_INSTALL_DIR/lib/*.* 3rdParty/adept/lib/
-  cp -P $DYNAWO_SUITESPARSE_INSTALL_DIR/lib/*.* 3rdParty/suitesparse/lib/
+  mkdir -p lib
+
+  cp -P $DYNAWO_SUNDIALS_INSTALL_DIR/lib*/*.* lib/
```

Comment:
Why do we need to put everything in the same folder?

Reply:
I find it way simpler and it is done the same way linux hierarchy folder is done. Moreover for mac we would need to add every path for each for the runtime in each binary as here only one path is enough.